onvm/go.sh

```diff
@@ -48,13 +48,32 @@ fi
 
 shift 3
 
+# Verify that bc is installed
+if [[ $(dpkg-query -W -f='${Status}' bc 2>/dev/null | grep -c "ok installed") == 0 ]]
```

Comment:
This is very Debian/Ubuntu specific. Would something simpler like checking the result of `which bc` also work?

Reply:
Changed that in the most recent commit. ShellCheck recommends using `command -v`, so I have that. Hopefully that works.